frontend/public/components/utils/dropdown.jsx

```diff
@@ -642,47 +642,104 @@ Dropdown.propTypes = {
   dataTest: PropTypes.string,
 };
 
-class ActionsMenuDropdown_ extends DropdownMixin {
-  render() {
-    const { actions, title = undefined, t } = this.props;
-    const onClick = (event, option) => {
-      event.preventDefault();
+const ActionsMenuDropdown_ = (props) => {
+  const { t } = useTranslation();
+  const [active, setActive] = React.useState(!!props.active);
+
+  const dropdownElement = React.useRef();
+
+  const show = () => {
+    setActive(true);
+  };
+
+  const hide = (e) => {
+    e && e.stopPropagation();
+    setActive(false);
+  };
 
-      if (option.callback) {
-        option.callback();
+  const listener = React.useCallback(
+    (event) => {
+      if (!active) {
+        return;
       }
 
-      if (option.href) {
-        history.push(option.href);
+      const { current } = dropdownElement;
+      if (!current) {
+        return;
       }
 
-      this.hide();
+      if (event.target === current || (current && current.contains(event.target))) {
```

Comment:
We've already checked that current is defined on L667, so we don't need to check again here:
```suggestion
      if (event.target === current || current.contains(event.target)) {
```